flutter-for-ios.md

```diff
@@ -27,3 +21,1847 @@ that are most relevant to your needs.
+HTTP call then you would use a `StatefulWidget`. After the HTTP call had
+complete, you would tell the Flutter framework that the widget’s `State` has
+been updated so it could update the widget.
+
+The important thing to note here is at the core both stateless and stateful
+widgets behave the same i.e. they rebuild every frame. The difference is that
+the `StatefulWidget` has a `State` object which stores state data across
+frames and restores it after being rebuilt.
+
+If you are in doubt, then always remember this rule: if a widget changes (e.g.
+because of runtime user interactions) it’s stateful. If the only time data
+changes is at instantiation, it’s stateless. However, if a widget is reacting
+to change, then the containing parent widget can still be stateless if it
+isn’t itself reacting to change.
+
+Let's take a look at how you would use a `StatelessWidget`. A common
+`StatelessWidget` is the `Text` widget. If you look at the implementation of
+the `Text` widget you'll find it subclasses `StatelessWidget`.
+
+<!-- skip -->
+{% prettify dart %}
+new Text(
+  'I like Flutter!',
+  style: new TextStyle(fontWeight: FontWeight.bold),
+);
+{% endprettify %}
+
+"If you look at the code above, you might notice that the `Text` widget
+"carries no explicit state with it. It renders what is passed in its
+"constructors and nothing more.
+
+But, what if you want to make "I Like Flutter" change dynamically, for example
+when clicking a `FloatingActionButton`?
+
+This can be achieved by wrapping the `Text` widget in a `StatefulWidget` and
+updating it when the button is clicked.
+
+For example:
+
+<!-- skip -->
+{% prettify dart %}
+class SampleApp extends StatelessWidget {
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Sample App',
+      theme: new ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: new SampleAppPage(),
+    );
+  }
+}
+
+class SampleAppPage extends StatefulWidget {
+  SampleAppPage({Key key}) : super(key: key);
+
+  @override
+  _SampleAppPageState createState() => new _SampleAppPageState();
+}
+
+class _SampleAppPageState extends State<SampleAppPage> {
+  // Default placeholder text
+  String textToShow = "I Like Flutter";
+  void _updateText() {
+    setState(() {
+      // update the text
+      textToShow = "Flutter is Awesome!";
+    });
+  }
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      appBar: new AppBar(
+        title: new Text("Sample App"),
+      ),
+      body: new Center(child: new Text(textToShow)),
+      floatingActionButton: new FloatingActionButton(
+        onPressed: _updateText,
+        tooltip: 'Update Text',
+        child: new Icon(Icons.update),
+      ),
+    );
+  }
+}
+{% endprettify %}
+
+## How do I layout my widgets? Where is my Storyboard? 
+
+In iOS, you may use a Storyboard file to organise your views and set
```

Comment:
According to the Google Style  Guide, our docs use American spelling, not British:

organise => organize

Reply:
Sorry, must've slipped through (Brits here!)